class-vip-support-user.php

```diff
@@ -400,6 +418,62 @@ public function filter_wp_redirect( $location ) {
 		return $location;
 	}
 
+	/**
+	 * Filters a users email address, removing the username if the email
+	 * is a VIP Support email alias.
+	 *
+	 * This helps simplify support email aliases in the users list so
+	 * they're not as lengthy and easier to understand at a glance.
+	 *
+	 * @param $email string The email address of a user.
+	 *
+	 * @return string
+	 */
+	public function filter_vip_support_email_aliases( $email ) {
+		if ( is_admin() && $this->is_vip_support_email_alias( $email ) ) {
+			return self::VIP_SUPPORT_EMAIL_ADDRESS;
+		}
+		return $email;
+	}
+
+	/**
+	 * Filters a users Gravatar URL, altering it if the email is a VIP
+	 * Support email alias.
+	 *
+	 * Since the email aliases are fake, they don't have a Gravatar.
+	 * Instead of showing the mystery man, get the Gravatar for the
+	 * real `VIP_SUPPORT_EMAIL_ADDRESS` email address.
+	 *
+	 * @param $url         string The Gravatar url.
+	 * @param $id_or_email mixed  The users ID, email, or a WP_User, WP_Post, WP_Comment object.
```

Comment:
Says that it could be a WP_Post or a WP_Comment, but there doesn't seem to be any code to support that here.